server/startup/migrations/v160.js

```diff
@@ -0,0 +1,21 @@
+import { Migrations } from '../../../app/migrations/server';
+import { Settings } from '../../../app/models/server';
+
+Migrations.add({
+	version: 160,
+	up() {
+		const desktopNotifications = Settings.findOne({ _id: 'Accounts_Default_User_Preferences_desktopNotifications' });
+		const mobileNotifications = Settings.findOne({ _id: 'Accounts_Default_User_Preferences_mobileNotifications' });
+
+		if (desktopNotifications && desktopNotifications.value === 'mentions') {
+			Settings.update({ _id: 'Accounts_Default_User_Preferences_desktopNotifications' }, { value: 'all' });
+		}
+
+		if (mobileNotifications && mobileNotifications.value === 'mentions') {
+			Settings.update({ _id: 'Accounts_Default_User_Preferences_mobileNotifications' }, { value: 'all' });
+		}
+	},
```

Comment:
I think this feels like this is going to be such a shocker for existing deployments.  :(

Reply:
hopefully for better :grimacing:

Reply:
same 😬

Reply:
We could use the new helper to alert admins about this change (if the setting was updated to `all`)

Reply:
I couldn't get a good enough looking banner to show to admins :/